File: decine_java/src/decine/UsuarioVIP.java
/*  
 *  Fundamentos de Ingenieria del Software
 *  Grado en Ingeniería Informática
 * 
 *  2014 © Copyleft - All Wrongs Reserved
 *
 *  Ernesto Serrano <info@ernesto.es>
 *  
 *  
 * 
 */
package decine;

public class UsuarioVIP {
    private String nombre;
    private String correo;
    
    UsuarioVIP(String nombre, String correo){
        this.nombre = nombre;
        this.correo = correo;
    }
    
    String getNombre() {
        return this.nombre;
    }
}

File: decine_ruby2/lib/usuario_vip.rb
# encoding: UTF-8
# 
#   Fundamentos de Ingenieria del Software
#   Grado en Ingeniería Informática
#
#   2014 © Copyleft - All Wrongs Reserved
# 
#   Ernesto Serrano <erseco@correo.ugr.es>
#   
#   
#

module Decine
  class UsuarioVip
    attr_reader :nombre, :correo
    def initialize(nombre, correo)
      @nombre = nombre
      @correo = correo
    end
  end
end
